src/debugpy/common/messaging.py

```diff
@@ -407,6 +426,13 @@ def wrap(self, key, *args, **kwargs):
     del _invalid_if_no_key
 
 
+class AssociatableMessageDict(MessageDict):
```

Comment:
I believe the correct word (can be associated) is "associable".

Reply:
Okay, will change.